lib/cassava/client.rb

```diff
@@ -0,0 +1,166 @@
+module Cassava
+  class Client
+    attr_reader :session
+
+    # @param [Cassandra::Session] The session object
+    def initialize(session)
+      @session = session
+    end
+
+    def insert_async(table, data)
+      session.execute_async(insert_statement(table, data), :arguments => data.values)
+    end
+
+    def insert(table, data)
+      session.execute(insert_statement(table, data), :arguments => data.values)
+    end
+
+    # @param [Symbol] table the table name
+    # @param [Array<Symbol>] An optional list of column names (as symbols), to only select those columns
+    # @return [StatementBuilder] A select builder representing the partially completed statement.
+    def select(table, columns = nil)
+      StatementBuilder.new(session).select(table, columns)
+    end
+
+    def delete(table, columns = nil)
+      StatementBuilder.new(session).delete(table, columns)
+    end
+
+    private
+
+    def insert_statement(table, data)
+      column_names = data.keys
+      statement_cql = "INSERT INTO #{table} (#{column_names.join(', ')}) VALUES (#{column_names.map { |x| '?' }.join(',')})"
```

Comment:
It's not obvious to me whether or not this is an issue at all, but I think it's a really good question.

* Insert is using a prepared statement, so it should be fine.
* This does not work:
```
cl.session.execute('SELECT * from test; SELECT * from test')
Cassandra::Errors::SyntaxError: line 1:20 missing EOF at 'SELECT'
```

That said, it would probably be preferrable for all statements to be prepared statements.